flexget/plugins/input/npo_watchlist.py

```diff
@@ -267,6 +270,8 @@ def _parse_episodes_std(self, task, config, series_name, page, series_info):
             e['npo_name'] = series_info['npo_name']
             e['npo_description'] = series_info['npo_description']
             e['npo_language'] = series_info['npo_language']
+            if subtitle[1]:  # if runtime is available in the subtext
```

Comment:
That will still fail. No matter what precedes `l[1]` it is a direct access to the 2nd element in the list.
```
>>> if l[1]:
...     pass
...
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
IndexError: list index out of range
>>>
```